src/bundle/Resources/config/routing.yml

```diff
@@ -342,3 +342,9 @@ ezplatform.search:
     methods: ['GET']
     defaults:
         _controller: 'EzPlatformAdminUiBundle:Search:search'
+
+ezplatform.content.set_main_location:
```

Comment:
Declaration is alright but placement in the file not so much. As you can see above in the same file I've tried to keep it organized using `# Comments` to separate sections. Can you place your route declaration in more appropriate section? Might be at the end of `# LocationView / Locations tab` section.